src/ipyradiant/visualization/cytoscape.py

```diff
@@ -7,40 +7,33 @@
 from rdflib import Graph, Literal, URIRef
 from rdflib.namespace import RDF
 
+from .base import VisBase
 
-class CytoscapeVisualization(W.GridBox):
-    graph = T.Instance(Graph, allow_none=True)
+
+class CytoscapeVisualization(VisBase):
```

Comment:
Docs here too plz

Reply:
Also, this PR includes cytoscape visualization updates and we never added an example notebook and cytoscape tests to the initial visualization PR. Can you please add a notebook test and example notebook for cytoscape (separate from Datashader)? You can make a separate branch and PR for the cytoscape tests if you want to get this merged faster.